packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -56,63 +84,28 @@ jest.mock('../../commerce-api/ocapi-shopper-baskets', () => {
     }
 })
 
-jest.mock('commerce-sdk-isomorphic', () => {
-    const sdk = jest.requireActual('commerce-sdk-isomorphic')
-    return {
-        ...sdk,
-        ShopperCustomers: class ShopperCustomersMock extends sdk.ShopperCustomers {
-            async getCustomerBaskets() {
-                return {baskets: [mockedBasketResponse]}
-            }
-        },
-        ShopperProducts: class ShopperProductsMock extends sdk.ShopperProducts {
-            async getProducts() {
-                return {data: [mockVariant]}
-            }
-            async getProduct() {
-                return {
-                    ...mockVariant,
-                    id: '750518699660M',
-                    variationValues: {
-                        color: 'BLACKFB',
-                        size: '050',
-                        width: 'V'
-                    },
-                    c_color: 'BLACKFB',
-                    c_isNew: true,
-                    c_refinementColor: 'black',
-                    c_size: '050',
-                    c_width: 'V'
-                }
-            }
-        },
-        ShopperPromotions: class ShopperPromotionsMock extends sdk.ShopperPromotions {
-            async getPromotions() {
-                return {
-                    count: 1,
-                    data: [
-                        {
-                            calloutMsg: "10% off men's suits with coupon",
-                            details: 'exceptions apply',
-                            id: '10offsuits',
-                            name: "10% off men's suits"
-                        }
-                    ],
-                    total: 1
-                }
-            }
-        }
-    }
-})
-
```

Comment:
This is beautiful ✨ ... love seeing a net -50% LOC (lines of code) change 🏅 
<img width="140" alt="image" src="https://user-images.githubusercontent.com/47546998/215582270-fbe713cd-9d93-409e-a34d-aff9ffbae501.png">